a.css

```diff
@@ -708,14 +734,19 @@ textarea {
   }
 
   .ulnav {
+    left: 0;
+    padding: 0;
+    width: 0;
+    height: 0;
     display: block;
     display: flex;
     list-style-type: none;
-    margin-top: 26px;
+    margin-top: -23px;
     margin-left: 860px;
```

Comment:
- [ ] The style in the design is a little different from the one you implemented. Kindly fix it.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/167716593-d7e3dd96-e9d6-4a42-9cef-e94e2775da00.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/167716723-3ff23727-7bea-478f-b105-3fe15730a191.png)